src/main/java/frc/robot/subsystems/PoseEstimator.java

```diff
@@ -130,17 +134,14 @@ private void useBestCamera() {
 
         if (cam2numTags > cam1numTags) {
             currentCamera = camera2;
-            robotToCamera = robotToTagCam2;
+            currentPhotonEstimator = photonEstimator2;
         } else if (cam2AvgDist < cam1AvgDist) {
             currentCamera = camera2;
-            robotToCamera = robotToTagCam2;
+            currentPhotonEstimator = photonEstimator2;
         } else {
             currentCamera = camera1;
-            robotToCamera = robotToTagCam1;
+            currentPhotonEstimator = photonEstimator1;
         }
```

Comment:
Perhaps add `&& cam2numTags == cam1numTags` to the else if to make sure we prioritize the camera with the most visible targets.

Reply:
I agree, done.